### ISSUE TITLE
Design support for new employee onboarding materials and swag

### ISSUE THREAD

**Issue by MandyRawls** [MozillaFoundation/plan#833]
Requesting design support for our new employee onboarding project, with the goal of making a more standardized, cohesive (and fun!) onboarding experience for all new employees.

Key pieces that require design support:
- creating a visual onboarding "roadmap"
- design for welcome package, including: welcome card, MoFo specific swag, etc. 
  
  A bit more detail has been provided to @xmatthewx 

Deadline: October 15th

**Comment by hannahkane** [comment]
@xmatthewx - are you able to provide an estimate for this?

Thanks for the ticket, @MandyRawls. Does the deadline need to include production time for the swag?

**Comment by xmatthewx** [comment]
- **Roadmap** – I would estimate 2 days for an info-graphic that visualizes the onboarding roadmap that could sit as an image on the foundation wiki. A bit more if it's a standalone pdf with branding. 
- **Card** – Welcome card (like a hallmark card?) could be 2 half days with feedback in between. 
- **Swag** – Swag is just a few hours of work if we have clear specs and just plan to use a logo. Planning swag, gathering specs, or designing custom artwork would be a couple of days. We should make an awesome fun t-shirt!

I'd suggest that a designer and Mandy meet next week to clarify requirements.

I'm probably under-estimating all of this ... small projects with fun potential can easily balloon without a lot of diligence. If the deadline includes production, we'd need to put swag on the fast track. 

It would be a challenge to get a contractor secured in time for that deadline (though we do have a good Toronto-based design who has worked with MoCo that could be a good candidate).

**Comment by xmatthewx** [comment]
@taisdesouzalessa – will see if her and @MandyRawls can meet this week to gather details on the requirements.

**Comment by xmatthewx** [comment]
Questions:
- [x] Is Oct 15th the deadline for design or printing?
- [x] Do we want original artwork for swag, t-shirt? Or just application of logo and text?
- [x] Where does the roadmap live? Wiki? PDF? Can it be high level, or will it require periodic updates?
- [x] others....?

**Comment by taisdesouzalessa** [comment]
Besides Matthew questions: 
- [x] will they provide content for the roadmap?
- [x] any example of roadmap/swag stakeholders like/find interesting?
- [x] How do you want the new employees feel (3 keywords)? Should those pieces show a bit of Mozilla Foundation culture (3 keywords)? That helps to define the style/theme of illustrations.

**Comment by taisdesouzalessa** [comment]
Here are the notes from my meeting with @MandyRawls . Mandy, if there is anything I missed or if you have any additional info, please add to the document: https://docs.google.com/a/mozilla.com/document/d/1VR1gyIGtihfkZUl5J-Wg5h5QglVJ8wqx5CPvP2BmtWc/edit?usp=sharing

**Comment by xmatthewx** [comment]
This looks great. Thanks @taisdesouzalessa.

I'll discuss with @hannahkane and make a plan.  

@MandyRawls – I would like to note that mid-October is pretty soon with MozFest in high gear. Two weeks could get us thru planning, gathering precedents, exploring design options, choosing a final direction. But we might need some time post Mozfest to wrap up.

**Comment by MandyRawls** [comment]
@xmatthewx - that timeline works for us. Please let me know if you need anything else. Thanks!

**Comment by MandyRawls** [comment]
I've updated the design document with the keywords for the feel/culture piece that Tais requested.

**Comment by flukeout** [comment]
**Some Questions**
- Which of these items should I focus on first? 
  - Roadmap, card, swag?
- The roadmap feels the most important and useful, but I don't see any content for it anywhere int eh linked docs - is it available somewhere?

I can get started on an illustration for the card and/or tshirt while the roadmap content is being created - will you be in charge of that content @MandyRawls?

**Comment by MandyRawls** [comment]
@flukeout I will add the roadmap content to this google doc: https://docs.google.com/a/mozilla.com/document/d/1VR1gyIGtihfkZUl5J-Wg5h5QglVJ8wqx5CPvP2BmtWc/edit?usp=sharing

**Comment by flukeout** [comment]
Testing the waters here with video game splash-screen concept. You're a fox starting a game. Works pretty well for onboarding I think. Here's a rough take...

![image](https://cloud.githubusercontent.com/assets/25212/20402790/07712682-acb3-11e6-8d5b-11fb0e578c70.png)

In the style of something like adventure time ...

![image](https://cloud.githubusercontent.com/assets/25212/20402807/174e6470-acb3-11e6-98cb-01878aaeaa10.png)

Would make a cool tshirt. Thoughts?

**Comment by taisdesouzalessa** [comment]
Wow Luke...this is incredible! I love the concept of exploration and adventure. Plus the design of the shirt, based on your sketch, is so cool that people would be proud of wearing it. 

My only thought would be to add some elements in his backpack or in the background scenario that references the issues we fight for. Just to add more a Mozilla's Foundation personality beyond the fox, to reinforce the concept of "fuelling a movement for the health of the internet" mentioned by Mark yesterday.

**Comment by MandyRawls** [comment]
Thanks, Luke! I like the idea of starting an exciting adventure as a MoFo. Would be great to tie in different elements of our work, too, in an informative and collective way. Also works nicely with the roadmap idea. Adding @Michaelaukland to the thread for his input, as well.

**Comment by xmatthewx** [comment]
Nice. Don't push too hard with references to issues. Could get cluttered and become less graphic, less punchy. 

Maybe instead of a shovel, fox carries a flag? For Internet? Open? One web? Keep it weird? Or is there a more webby tool?

**Comment by flukeout** [comment]
How else will the fox **dig** into issues @xmatthewx, it's gotta stay

**Comment by flukeout** [comment]
I like flag idea though.

**Comment by flukeout** [comment]
Here's what I got so far

![image](https://cloud.githubusercontent.com/assets/25212/20407304/9577e3e6-acc5-11e6-9547-1a2d91dff0e2.png)

**Comment by flukeout** [comment]
Ok, one more update, I'll leave this here for a bit to get some feedback.

![image](https://cloud.githubusercontent.com/assets/25212/20411463/7433d526-acd6-11e6-9fbe-0b67b5bc6722.png)

**Comment by MandyRawls** [comment]
@flukeout Michael and I were chatting, and love the adventure starting aspect of the design for the onboarding experience (esp the roadmap), but are think it may be better to have more generic MoFo branding for the swag - possibly using the Mozilla branding but making it more MoFo. This way the swag is a bit more inclusive (any MoFo would be proud to wear or use it at anytime, during or after their onboarding). Is there any MoFo branding that already exists, that we can use as a starting point?

**Comment by flukeout** [comment]
Hey @MandyRawls - the only real 'style' mofo has are the line-art icons that are used on the Learning Site, like these...

![image](https://cloud.githubusercontent.com/assets/25212/20409530/d91f5616-acce-11e6-9707-ccec5fd459b9.png)

![image](https://cloud.githubusercontent.com/assets/25212/20409539/e26c91ac-acce-11e6-849c-a38c5fb79567.png)

Other than that, it just bold blocks of color and text. We've also got some patterns, as seen on the thimble homepage for example...

![image](https://cloud.githubusercontent.com/assets/25212/20409568/fca08844-acce-11e6-8b85-87e5f8ea3f1f.png)

Which specific swag items do you think would be the best?

**Comment by jgmac1106** [comment]
FWIW Would it be better to wait on a long term onboarding of SWAG as the
Mozilla redesign effort is underway?

The community review is underway but you folks would know the timeline
better than me.

If you want more generic MoFo style you could just copy the Mozilla Clubs
logo style guide.

Though I want this @flukeout in both shirt and sticker. Design total keeper

**Comment by xmatthewx** [comment]
@MandyRawls – The Mozilla logo and brand is likely to change considerably in the not too distant future. MozFest is probably the most established MoFo-y identity. 

We could do a shirt with the current Mozilla logo backed by something more MoFo-ish, but again this will change soon. On the other hand, I don't think a card needs to be as closely coupled with the brand. Much like our campaign sites and materials, they can have a bit more unique character.

I suggest that you Michael and Luke discuss over vidyo to line up, before pushing this further.

**Comment by flukeout** [comment]
We could use some of our icons as the imagery, I've picked out three that could work well. What do you think of a direction like this?

![image](https://cloud.githubusercontent.com/assets/25212/20414064/74e987cc-ace5-11e6-95a4-24f979f093fc.png)

We could do a logo on front with the Mozilla Foundation wordmark on the back.

**Comment by flukeout** [comment]
An update cc @MandyRawls @Michaelaukland - let me know your thoughts and ping me on mattermoz if you'd like to chat, schedule a call, etc.

![image](https://cloud.githubusercontent.com/assets/25212/20439888/98e93e5a-ad72-11e6-821f-c0ea1d554250.png)

**Comment by flukeout** [comment]
![image](https://cloud.githubusercontent.com/assets/25212/20447342/2d94f790-ad93-11e6-970f-2d669e2af29f.png)

**Comment by flukeout** [comment]
![image](https://cloud.githubusercontent.com/assets/25212/20450003/dd0fe208-ada1-11e6-9cb1-59ae42845427.png)

![image](https://cloud.githubusercontent.com/assets/25212/20450241/b804bb30-ada3-11e6-9b72-c83ec4c6593e.png)

![image](https://cloud.githubusercontent.com/assets/25212/20450317/5d158690-ada4-11e6-8cf2-f5df8fb1b1bc.png)

![image](https://cloud.githubusercontent.com/assets/25212/20450014/f3d3f48e-ada1-11e6-9a7b-547822c75088.png)

**Comment by flukeout** [comment]
Could try a "foundation" kind of intertwined through the bold letters saying Mozilla, like here...
![image](https://cloud.githubusercontent.com/assets/25212/20450860/a96470c0-ada8-11e6-8984-4eff9b4ef41c.png)

**Comment by flukeout** [comment]
Maybe a "Crazy Mofo" in this style..

![image](https://cloud.githubusercontent.com/assets/25212/20450941/3f3f9fb6-ada9-11e6-83a8-73f0d227c36c.png)

**Comment by flukeout** [comment]
Do we have a manifesto or something like that? A chunk of text that we can use for a more typographic design?

**Comment by MandyRawls** [comment]
@Lottarao , @Michaelaukland , Luke and I have a meeting set for Wednesday to discuss.

**Comment by flukeout** [comment]
Based on the latest meeting, we've agreed to go with an illustrated or typographical version of "mofo/MOFO/MoFo" for the shirt, sticker and bottle and potentially using accent colors from our mofo-bootstrap color palette.

We'll tackle this first, then move to the onboarding materials. I'll post design ideas and updates in this issue soon.

**Comment by flukeout** [comment]
Okay so, I've got some [sketches in this here deck](https://docs.google.com/presentation/d/1Zzko82kIjRNxxfX_ug831JA7B7Zi7QKlAAVDFQq01O0/edit#slide=id.p). 

In talking with the team, I think we're all in agreement that just "Mofo" wont cut it, and it might be offensive so... we're gonna go with some kind of type treatment of Mozilla Foundation.

Just getting rolling here. cc @MandyRawls @Lottarao

**Comment by MandyRawls** [comment]
Sounds good - thanks @flukeout !

**Comment by flukeout** [comment]
[Updated deck with some more ideas](https://docs.google.com/presentation/d/1c__DKKIJmDdmKvZPOhz50OyPJbYCIkESqzYZBxrzPnw/edit#slide=id.g197cb5db5a_0_59)

cc @MandyRawls Let me know your thoughts on these directions. Cheers.

**Comment by flukeout** [comment]
@MandyRawls a few more updates today, specifically the concept on slides 10-12 - just a heads up before you meet about it tomorrow. Cheers.

**Comment by MandyRawls** [comment]
@flukeout is it possible to edit slide 12 to make the design coloured (like slide 10)? We would like to share that slide with the team and propose we move ahead with that design for the swag. Thank you!

**Comment by flukeout** [comment]
It certainly is @MandyRawls - I'll create a new deck for that along with the new bottle. However, I think we should consider the greyscale version too - the consensus from the esteemed design team is that's the option they'd rather wear.

**Comment by MandyRawls** [comment]
@flukeout sounds good! Can we have two updated slides then - one for the coloured version and one for the greyscale version? We will present both options. Thanks!

**Comment by flukeout** [comment]
@MandyRawls as promised - [here is the updated deck](https://docs.google.com/a/mozilla.com/presentation/d/186kx2LsKqejR8pVvjpmGxsVXIeZFJuaOq7aP09t8Msc/edit?usp=sharing). Cheers!

**Comment by flukeout** [comment]
Following up from All-hands after chatting with @MandyRawls 

* **Swag** - We are stopping work on the swag designs and waiting for the new mozilla branding to kick in. Once that settles we'll see how we fit the foundation brand in there.
  * RIP cool logo - https://flukeout.github.io/foundation-swag/
* **Onboarding** - Will continue working on this...
  * Will build as a web page
  * Will use either the 'game-theme' with dapper fox or an adaptation of the hand-drawn swag style

Sounds accurate? I'll close this and create a new issue to tackle the onboarding.

**Comment by xmatthewx** [comment]
An onboarding page on github? Could this be a page on mofo.build, for easier updates?

**Comment by flukeout** [comment]
That could work - I'll look into how that stuff is made & maintained.

**Comment by MandyRawls** [comment]
We are good with either a web page or PDF doc - whatever is easiest to create and maintain. Note that if we do go with a public web page, we will be linking to some sites that are Mozilla internal only.  

It will basically work as both an exciting/welcoming overview of what the first 6 months of employment at MoFo will look like, and as a landing page for new hires (each milestone will link to additional resources). We like the idea of a game board type road map with the dapper fox and simple graphics. 

Please let me know if you need anymore info!
